lib/ios-sim.ts

```diff
@@ -8,7 +8,7 @@ function getSimulator(): ISimulator {
 
 const publicApi = {};
 
-Object.defineProperty(publicApi, "getRunningSimulator", {
+Object.defineProperty(publicApi, "getRunningSimulators", {
```

Comment:
why is this method name changed? It still returns a single booted device. Also removing method from Public API will require increment of the major version of the package. I suggest to remove this change and introduce a new method - `getRunningSimulators` that returns multiple simulator instances (i.e. call `getBootedDevices` instead of `getBootedDevice`. After that we can use the new method in CLI.

Reply:
Because we returns array of booted devices, not just the one booted device. I agree to revert this change and introduce new method :+1: